lib/IRGen/IRGenModule.cpp

```diff
@@ -1776,3 +1777,45 @@ TypeExpansionContext IRGenModule::getMaximalTypeExpansionContext() const {
 const TypeLayoutEntry &IRGenModule::getTypeLayoutEntry(SILType T) {
   return Types.getTypeLayoutEntry(T);
 }
+
+/// Return whether FrameLowering should always set the "extended frame
+/// present" bit in FP, or set it based on a symbol in the runtime.
+static bool isSwiftAsyncContextIsDynamicallySet(llvm::Triple TT) {
```

Comment:
Can you use `isConcurrencyAvailable` from `GenMetadata.cpp` rather than hard-coding these values again?